eval/src/vespa/eval/eval/value_type.h

```diff
@@ -17,6 +16,7 @@ class ValueType
 {
 public:
     enum class Type { ANY, ERROR, DOUBLE, TENSOR };
+    enum class CellType {FLOAT, DOUBLE};
```

Comment:
a 0-dimension tensor with 32-bit precision would now be a 'DOUBLE' with cell type 'FLOAT', which kind of degenerates to a number like before, except it is not supported in compiled expressions.

Reply:
Reverted as discussed